tests/test_tasks.py

```diff
@@ -56,20 +56,16 @@ class TestTasksPost(EndpointTestCase):
 
     # Fixture specific URL to available student data.
     VALID_URL = "/api/students/2/tasks/"
-    # Fixture specific URL to student data not available for current user.
+    # Fixture specific URL to data not available for current user.
     NOT_PERMITTED_URL = "/api/students/1/tasks/"
     # Fixture specific URL to invalid student ID.
     TASK_NOT_FOUND_URL = "/api/students/12345/tasks/"
 
     # Valid task data.
-    VALID_JSON_TASK_POST = {
-        "name": "Another new task to do",
-        "value": 21,
-    }
+    VALID_TASK_DATA = {"student": "2", "name": "Another new task to do", "value": 21}
```

Comment:
or that propose:
    # Valid task data.
    VALID_STUDENT_ID = {"student_id": "2"}
    VALID_TASK_DATA = {"name": "Another new task to do", "value": 21}

    def test_Success(self):
        response = self.post(self.VALID_URL, self.VALID_TASK_DATA)

        # General assertions.
        assert response.status_code == status.HTTP_201_CREATED
        assert response.headers["Content-Type"] == "application/json"
        
        # Fixture specific assertions.
        single_task_url = "/api/students/"+self.VALID_STUDENT_ID["student_id"]+"/task/3/"
        post_op_data = self.get(single_task_url).json()
        assert post_op_data["name"] == "Another new task to do"
        assert post_op_data["value"] == 21


because - on szwagier:
![params123](https://github.com/user-attachments/assets/a01c18e7-31d1-4e84-837e-005d72b5435f)

Reply:
Okay, removed student field.